lib/zone.ts

```diff
@@ -323,6 +329,8 @@ interface _ZonePrivate {
   patchEventTargetMethods:
       (obj: any, addFnName?: string, removeFnName?: string, metaCreator?: any) => boolean;
   patchOnProperties: (obj: any, properties: string[]) => void;
+  beforeRunTask: (zone: Zone, task: Task) => BeforeRunTaskStatus;
```

Comment:
yes, those APIs are private.

And I will check `fakeAsync` , I think the `runTask` will be called because in `fakeAsyncZone`, the callback is `task.invoke` and it will finally call `task.zone.runTask`. I will check it.